platform/server/elasticsearch/__tests__/ScopedDataClient.test.ts

```diff
@@ -0,0 +1,58 @@
+const mockCallAPI = jest.fn();
```

Comment:
__note:__ I was about to say that we should probably clear that global mock in the `afterEach` to avoid any potential subtle issues even though we don't have any now and then I've just noticed that global Jest configuration has [`clearMocks`](https://facebook.github.io/jest/docs/en/configuration.html#clearmocks-boolean) option that would eliminate my concerns, but it's disabled by default for some reason .

Do you know any reasons why we shouldn't enable this option for Kibana @kjbekkelund @archanid ?

Reply:
I think it's worth enabling `clearMocks` and seeing what happens.